server/db/MockDB.go

```diff
@@ -0,0 +1,114 @@
+package db
+
+import (
+	"errors"
+
+	"github.com/google/uuid"
+	"github.com/jak103/uno/model"
+)
+
+// MockDB is an implemenation declaring the unit test db
+type mockDB struct {
+	games         map[string]model.Game
+	gamePasswords map[string]model.Game
+	players       map[string]model.Player
+}
+
+//type mongoDB struct {
+//	client   *mongo.Client
+//	uri      string
+//	database *mongo.Database
+//	games    *mongo.Collection
+//	players  *mongo.Collection
+//}
+
+// HasGame checks to see if a game with the given ID exists in the database.
+func (db *mockDB) HasGameByPassword(password string) bool {
+	_, ok := db.gamePasswords[password]
+	return ok
+}
+
+// HasGameByID checks to see if a game with the given ID exists in the database.
+func (db *mockDB) HasGameByID(id string) bool {
+	_, ok := db.games[id]
+	return ok
+}
+
+// CreateGame a game with the given ID. Perhaps this should instead just return an id?
+func (db *mockDB) CreateGame() (*model.Game, error) {
+	myGame := model.Game{ID: uuid.New().String(), Password: "12234"}
```

Comment:
Why is the password hardcoded?

Reply:
This is a mockDB. it's specifically to be used for unit tests. The Game ID may be hardcoded at some point depending on how unit tests go long term.